titus-server-master/src/main/java/com/netflix/titus/master/appscale/service/DefaultAppScaleManager.java

```diff
@@ -167,6 +176,12 @@ public Completable enterActiveMode() {
         checkForScalingPolicyActions().toCompletable().await(appScaleManagerConfiguration.getStoreInitTimeoutSeconds(),
                 TimeUnit.SECONDS);
 
+        reconcileAllPendingRequests = Observable.interval(appScaleManagerConfiguration.getReconcileAllPendingAndDeletingRequestsIntervalMins(), TimeUnit.MINUTES)
```

Comment:
`ObservableExt.schedule` does this, instruments the code, and handles error cases.